docker/Dockerfile

```diff
@@ -0,0 +1,43 @@
+ARG DOTNET_VERSION=6.0
+FROM mcr.microsoft.com/dotnet/runtime-deps:${DOTNET_VERSION}-alpine
+
+# Copy entrypoint script into container
+COPY ./entrypoint.sh /entrypoint.sh
+
+ARG VERSION=latest
+ARG TARGETPLATFORM
+ARG REPOSITORY=rcdailey/trash-updater
+RUN set -ex; \
+    \
+    # The download path is a bit different when using the latest release instead of a tagged release
+    if [ "$VERSION" = "latest" ]; then \
+        download_path="latest/download"; \
+    else \
+        download_path="download/$VERSION"; \
+    fi; \
+    \
+    # Determine the runtime from the target platform provided by Docker Buildx
+    case "$TARGETPLATFORM" in \
+        "linux/arm/v7") runtime="linux-musl-arm" ;; \
+        "linux/arm64") runtime="linux-musl-arm64" ;; \
+        "linux/amd64") runtime="linux-musl-x64" ;; \
+        *) echo >&2 "ERROR: Unsupported target platform: $TARGETPLATFORM"; exit 1 ;; \
+    esac; \
+    \
+    # Download and extract the binary from the release
+    wget --quiet -O /tmp/trash.zip "https://github.com/$REPOSITORY/releases/$download_path/trash-$runtime.zip"; \
+    apk add --no-cache unzip; \
+    unzip -j /tmp/trash.zip -d /usr/local/bin trash; \
+    \
+    # Set permissions
+    chmod 755 /usr/local/bin/trash; \
+    chmod 755 /entrypoint.sh; \
+    \
+    # Cleanup system
+    apk del unzip; \
+    rm -rf /tmp/trash.zip; \
+    rm -rf /var/spool/cron/crontabs/root; \
+    rm -rf /etc/periodic
+
+# Define the entrypoint script as the entrypoint for the container
+ENTRYPOINT ["/entrypoint.sh"]
```

Comment:
If you stop this docker container gracefully (e.g. `docker stop` after `docker run -d`), does it time out and kill the process? PID 1 has some special requirements that will prevent scripts from exiting cleanly. I've always incorporated [tini](https://github.com/krallin/tini) in my Alpine images to remedy this.

Is this something you've tested?

Reply:
As of Docker version 1.13, tini is included in Docker itself and can be activated by specifying using `--init` argument. That's why I left it out of the image to reduce size (I actually used it initially).

Reply:
Yeah I read about that. That's news to me. I built up this habit of using `tini` years ago and never looked back. I'm a little disappointed that it requires an additional command line option. I personally feel like it should be a default. I won't ask that you change it since a command line option is present, though. Thanks for the info!